src/test/ui/issues/issue-15919-64.stderr

```diff
@@ -0,0 +1,8 @@
+error: the type `[usize; 18446744073709551615]` is too big for the current architecture
```

Comment:
The 64bit and 32bit output are the same except for this number, what is the trouble with using normalization? We used this successfully before.

Reply:
Oh wait the tests have different code. That seems odd. But they are still `ui` tests so whatever...

But please add a comment explaining why these need to be separate tests.

Reply:
They need to have different literals, we can't annotate with `//~ ERROR` on lines with conditional compilation, and the `stderr` output would point at different lines in the different runs. Where I could keep the same semantic meaning in a single file, I did so, like in `src/test/ui/huge-enum.rs`.